.env.sample

```diff
@@ -1,7 +1,7 @@
 # dev, local, brickable, harbour, mainnet
 CHAIN_VERSION=dev
 
-BOOTNODE_ID_LOCAL=QmWYmZrHFPkgX8PgMgUpHJsK6Q6vWbeVXrKhciunJdRvKZ
+BOOTNODE_ID_LOCAL=12D3KooWKS7jU8ti7S5PDqCNWEj692eUSK3DLssHNwTQsto9ynVo
```

Comment:
yeh i think it will work for 'dev' too, i will try now.
it's the same as in https://github.com/DataHighway-DHX/node/blob/master/node/src/chain_spec.rs#L258 so there's no conflict. 'dev' doesn't have one in the chain_spec.rs file

Reply:
ok, just making sure that ID works for both if we share that variable for dev and local

Reply:
yes it works for 'dev' too